irods/message/__init__.py

```diff
@@ -130,7 +137,7 @@ def ET(xml_type = 0, server_version = None):
     `server_version', if given, should be a list or tuple specifying the version of the connected iRODS server.
 
     """
-    if xml_type is not 0:
+    if xml_type is not _ZERO:  # "is not 0" printed SyntaxWarning upon initial byte-compilation
```

Comment:
does `!=` work instead of `is not`?

Reply:
for this purpose , no, because MyIntEnum (needed for Python2) is designed to support comparisons to integers.

Reply:
Soon as we remove Py2 support, this MyInt related code can go away! Yay 2.0.0